recipes/ocw/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ocw" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "8f12405ec02ea327b1e0cf65547e409232f67577be6eb348d2ea9011827e2c4a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - basemap
+    - netcdf4
+    - h5py
+    - bottle
+    - pydap
+    - python-dateutil
+    - mock
```

Comment:
Do we need `mock` at run time? Is this being used for something other than tests?